src/test/common/clara/test_memory.cljc

```diff
@@ -43,10 +44,48 @@
 
 (use-fixtures :once st/validate-schemas #?(:clj tu/opts-fixture))
 
+(defn join-filter-equals
```

Comment:
Seems like this could be added to `clara.tools.testing-utils` and then used in the (now) various testing ns's?